collapsibleTabs.js

```diff
@@ -1 +1 @@
collapsible.data("collapsibleTabsSettings");if(!settings){$.collapsibleTabs.addData($collapsible);settings=$collapsible.data("collapsibleTabsSettings")}return settings},handleResize:function(){$.collapsibleTabs.instances.each(function(){var $el=$(this),data=$.collapsibleTabs.getSettings($el);if(data.shifting){return}if($el.children(data.collapsible).length>0&&data.collapseCondition()){$el.trigger("beforeTabCollapse");$.collapsibleTabs.moveToCollapsed($el.children(data.collapsible+":last"))}if($(data.collapsedContainer+" "+data.collapsible).length>0&&data.expandCondition($.collapsibleTabs.getSettings($(data.collapsedContainer).children(data.collapsible+":first")).expandedWidth)){$el.trigger("beforeTabExpand");$.collapsibleTabs.moveToExpanded(data.collapsedContainer+" "+data.collapsible+":first")}})},moveToCollapsed:function(ele){var outerData,expContainerSettings,target,$moving=$(ele);outerData=$.collapsibleTabs.getSettings($moving);if(!outerData){return}expContainerSettings=$.collapsibleTabs.getSettings($(outerData.expandedContainer));if(!expContainerSettings){return}expContainerSettings.shifting=true;target=outerData.collapsedContainer;$moving.css("position","relative").css(rtl?"left":"right",0).animate({width:"1px"},"normal",function(){var data,expContainerSettings;$(this).hide();$('<span class="placeholder" style="display: none;"></span>').insertAfter(this);$(this).detach().prependTo(target).data("collapsibleTabsSettings",outerData);$(this).attr("style","display: list-item;");data=$.collapsibleTabs.getSettings($(ele));if(data){expContainerSettings=$.collapsibleTabs.getSettings($(data.expandedContainer));if(expContainerSettings){expContainerSettings.shifting=false;$.collapsibleTabs.handleResize()}}})},moveToExpanded:function(ele){var data,expContainerSettings,$target,expandedWidth,$moving=$(ele);data=$.collapsibleTabs.getSettings($moving);if(!data){return}expContainerSettings=$.collapsibleTabs.getSettings($(data.expandedContainer));if(!expContainerSettings){return}expContainerSettings.shifting=true;$target=$(data.expandedContainer).find("span.placeholder:first");expandedWidth=data.expandedWidth;$moving.css("position","relative").css(rtl?"right":"left",0).css("width","1px");$target.replaceWith($moving.detach().css("width","1px").data("collapsibleTabsSettings",data).animate({width:expandedWidth+"px"},"normal",function(){$(this).attr("style","display: block;");var data,expContainerSettings;data=$.collapsibleTabs.getSettings($(this));if(data){expContainerSettings=$.collapsibleTabs.getSettings($(data.expandedContainer));if(expContainerSettings){expContainerSettings.shifting=false;$.collapsibleTabs.handleResize()}}}))},calculateTabDistance:function(){var $leftTab,$rightTab,leftEnd,rightStart;if(!rtl){$leftTab=$("#left-navigation");$rightTab=$("#right-navigation")}else{$leftTab=$("#right-navigation");$rightTab=$("#left-navigation")}leftEnd=$leftTab.offset().left+$leftTab.width();rightStart=$rightTab.offset().left;return rightStart-leftEnd}}})(jQuery);
```

Comment:
What is changed in this file?

Reply:
Can we unminify this on master branch? Then I'll rebase.

Reply:
I've unminified it. You can see clearly the change (use of $.debounce).
This code is better unminified since presumably it will evolve with the Vector script it copies.

Reply:
Could you please rebase this PR? I've removed code minification from collapsibleTabs.js to see that was changed

Reply:
done